source/_patterns/01-molecules/components/highlight-item.yaml

```diff
@@ -0,0 +1,25 @@
+$schema: http://json-schema.org/draft-04/schema#
+type: object
+properties:
+  title:
+    type: string
+    minLength: 1
+  url:
+    type: string
+    minLength: 1
+  authors:
+    type: string
```

Comment:
`minLength: 1` still applies. If the value is present it should not be empty.

Reply:
Done